recipes/r-wicket/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = '0.4.0' %}
+
+package:
+  name: r-wicket
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: wicket_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/wicket_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/wicket/wicket_{{ version }}.tar.gz
+  sha256: ce76a4c8f79e7dfb9d8aadfa59b7f5cdb0de3dd9623b1426fd5caf6464fd3fd2
+
+build:
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - gcc
+    - r-bh
+    - r-rcpp
+
+  run:
+    - r-base
+    - libgcc         # [win]
```

Comment:
Could you get rid of the #[win] tag? Bioconda does not build for windows, so its not really applicabble.

Reply:
Ha yes indeed, some skeleton cran comments i forgot to remove